include/debug/tracing_core.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
files added in include/debug/ are generic and not related to ctf, split them out into their own commit.